src/display/editor/annotation_editor_layer.js

```diff
@@ -91,13 +94,45 @@ class AnnotationEditorLayer {
     this.#uiManager.addLayer(this);
   }
 
+  /**
+   * The mode has changed: it must be updated.
+   * @param {number} mode
+   */
+  updateMode(mode) {
+    if (mode === AnnotationEditorType.INK) {
+      // We want to have the ink editor covering all the page
+      // and no click to create it: it must be here when we start to draw.
+      this.div.addEventListener("mouseover", this.#boundMouseover);
+      this.div.removeEventListener("click", this.#boundClick);
+    } else {
+      this.div.removeEventListener("mouseover", this.#boundMouseover);
+    }
+  }
+
+  /**
+   * mouseover callback.
+   * @param {MouseEvent} event
+   */
+  mouseover(event) {
+    if (event.target === this.div && event.buttons === 0) {
+      // The div is the target so there is no ink editor, hence we can
+      // create a new one.
+      // event.buttons === 0 is here to avoid adding a new ink editor
+      // when we drop and editor.
```

Comment:
Did you actually mean the following?
```suggestion
      // when we drop an editor.
```